iatistandard/root/index.html

```diff
@@ -103,25 +103,25 @@ <h3 class="assistive-text">Main menu</h3>
             <!-- Breadcrumb NavXT 4.4.0 -->
 <!--<a title="Go to The IATI Standard." href="http://iatistandard.org" class="home">The IATI Standard</a> &gt; The IATI Activities Standard-->        </div><!--.breadcrumbs -->
         <div id="main">
-        
+
             <div id="home-strapline">
                 <h1>IATI Standard is a technical publishing framework allowing data to be compared. This site gives detail on the standard, how to implement it and provides guidance.</h1>
-                <h2>This website provides reference material for the <a href="202/upgrades/all-versions/">different versions of the IATI Standard</a>. If you do not actively choose a version, the links below will take you the latest 2.02 version.</h2>
+                <h2>This website provides reference material for the <a href="202/upgrades/all-versions/">different versions of the IATI Standard</a>. If you do not actively choose a version, the links below will take you the latest version (2.03).</h2>
```

Comment:
Suggest removing mention of `2.03` entirely as part of the "stop worrying about versions" concept.

Decimals should be backwards-compatible, so there's not really a need to mention an explicit version number here. Integers make it important to look at the correct docs, though specifying a Decimal Version doesn't really help convey anything about Integer changes.

Reply:
Fixed in c5baf64